library_garage/db/load_data_to_db.rb

```diff
@@ -0,0 +1,70 @@
+require '../classes/author.rb'
+require '../classes/book.rb'
+require '../classes/reader.rb'
+require '../classes/order.rb'
+require '../classes/library.rb'
+require 'yaml'
+
+class Downloader
```

Comment:
We discussed it on review. This class has no own data, it only used library and its data/methods, so you can't simply create library and fill it at any time. Downloader will create new library each time. Module will be better here.